packages/iov-bns/src/index.ts

```diff
@@ -1,2 +1,3 @@
 export { bnsCodec } from "./bnscodec";
 export { Client } from "./client";
+export { bnsFromOrToTag, bnsNonceTag } from "./tags";
```

Comment:
This is a good call.
It didn't belong in writer.ts at all, just wanted it to be exposed to the outside world through iov-core.

We can rethink how tags are built later, but this is a nice cleanup. step by step